race/SeiTime;Amalgatrix.json

```diff
@@ -0,0 +1,252 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"abbreviation": "AMIX",
+				"json": "AMIX Lineage",
+				"full": "Amalgatrix",
+				"authors": "Sei Time",
+				"url": "https://www.dndbeyond.com/races/1136727-amalgatrix",
+				"version": "2023-03-16"
+			}
+		]
+	},
+	"name": "Amalgatrix",
```

Comment:
This and the following properties should be within an object in the `race` array. So it should look like:
```json
"race": [
    {
        "name": "Amalgamatrix",
        "source": "AMIX",
        ...
```